src/lifecycle/created.js

```diff
@@ -55,21 +56,25 @@ function createCallUpdateOnProperties (opts) {
       if (prop && prop.type) {
         val = (prop.type === Boolean && elem.hasAttribute(typeof prop.attr === 'string' ? prop.attr : name)) || prop.type(val);
       }
-      update && update.call(elem, val);
+      update && update(elem, {
+        name: name,
+        newValue: val,
+        oldValue: null
```

Comment:
Yeah I'll fix that. Writing some unit tests for this stuff specifically right now.